api/pkg/db/model/model.go

```diff
@@ -33,6 +33,13 @@ type (
 		Resources []*Resource `gorm:"many2many:resource_tags;"`
 	}
 
+	Platform struct {
+		gorm.Model
+		Name             string             `gorm:"not null;unique"`
+		ResourceVersions []*ResourceVersion `gorm:"many2many:resource_platforms;"`
+		Resource         []*Resource        `gorm:"many2many:version_platforms;"`
```

Comment:
mismatch I guess :)
`version_platforms` for `Resource` instead of `resource_platform`

Reply:
Thank you and good catch :)